### PR TITLE
✨ collection totalItems & updatedAt

### LINKED ISSUE #128: Count NFTs in the Collection Entity
The goal of this task to have counts of the NFTs in the Collection entity:

In the `schema.graphql` you need to extend `CollectionEntity` with following fields 

```graphl
amount: Int!
total: Int!
```

where `total` would represent a total amount of NFTs in collection and `amount` a total amount of non-burned NFTs

next:
1. do a codegen!

2. In the `createCollection` function you need to add 
```ts
collection.amount = 0
collection.total = 0
```

3. In the `mint` function you need to add 
```ts
collection.amount += 1
collection.total += 1
```

3. In the burn function you need to get corresponding `collection`
```ts
const collection = await get(/* ...params */)
collection.amount -= 1
await context.store.save(collection)
```

*4. TEST!*

That should be all. 
Happy hacking.

Ref:
- https://github.com/ma-shulgin/rmrk2-integration/blob/main/schema.graphql#L85

_Originally posted by @roiLeo in https://github.com/kodadot/rubick/issues/127#issuecomment-1218066619_

### PR DESCRIPTION
**Pull request by roiLeo**
**Thank you for your contribution** to the [KodaDot Indexer](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [x] Feature

### What's new?

- [x] PR closes #128 
- [x] new `updatedAt` field in collection
- [x] new `totalItems` & `totalAvailableitems` field in collection
- [x] This PR is required for I don't remmeber but it's needed

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [x] I've run the indexer and it hasn't failed
- [x] I've changed schema and performed a migration

### Screenshot

Capture d’écran 2022-10-13 à 2 00 29 PM

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -27,6 +27,9 @@ type CollectionEntity @entity {
   blockNumber: BigInt
   meta: MetadataEntity
   createdAt: DateTime!
+  updatedAt: DateTime!
+  totalItems: Int!
+  totalAvailableItems: Int!
```
Comment (vikiival) [inline schema.graphql:32]:
```suggestion
  nftCount: Int!
  supply: Int!
```

**Review by vikiival — COMMENTED** [review]
Awesome thanks for the PR!.

Make sure to test it and I would presonally changed the name

File: schema.graphql (hunk 1 above)
Comment (roiLeo) [reply schema.graphql:32]: Can't commit your suggestion as we will clearly need to recreate a migration file
`nftCount` & `supply` feels meh :/ (from the perspective of a dev I find it hard to tell the difference) but I'll change that

**Comment by roiLeo** [comment]
✅ updatd

Capture d’écran 2022-10-17 à 12 28 35 PM

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
@roiLeo can you please run this query?

```graphql
query MyQuery {
  collectionEntities(limit: 5, where: {nfts_some: {burned_eq: true}}) {
    id
    nftCount
    supply
    nfts {
      id
      burned
    }
  }
}
```

**Comment by vikiival** [comment]
to check if we count it correctly

**Comment by roiLeo** [comment]
After a while:
Capture d’écran 2022-10-17 à 1 56 50 PM

Capture d’écran 2022-10-17 à 2 21 38 PM
Capture d’écran 2022-10-17 à 2 24 05 PM

something wrong? might come from `plsBe(real, ...)`

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
Will go out with milestone 008

**Pull request merged**